gcs_to_bq/common/census_to_bq.py

```diff
@@ -0,0 +1,72 @@
+import csv
```

Comment:
Is this where Jennie's household income stuff will go as well? Maybe we could put a TODO to add hers to this file or rename the file so it's clear how it's different than hers (since they're now both in the census file for ingestion)

Not sure if the two were meant to be in the same bigquery table..

Reply:
Could go either way. I'm going to add in a few more census sources here and when I do that I'll refactor this into a way that's more useful for other census data, I think that will make it clearer how Jennie's stuff will fit in. I think they'll all be separate BQ tables though.